build/configs/qemu/HowToRunNetworkStackOnQemu.md

```diff
@@ -0,0 +1,85 @@
+# How to run Network Stack on QEMU
+LWIP is the network stack supported for QEMU on TiznRT.  
+
+## Steps to verify network stack on QEMU is as follows
+
+Prerequisites: Connect WiFi AP to Linux machine(wired connection. dhcp should be enabled in the AP).  
```

Comment:
# How to run Network Stack on QEMU
LWIP is the network stack supported for QEMU on TiznRT. (Need to change)

## Prerequisites
Connect WiFi AP to Linux machine (wired connection. dhcp should be enabled in the AP).

## Steps to verify network stack

### 1. Build QEMU image on Host
```
cd $TIZENRT_BASEDIR/os/tools
./configure.sh qemu/tc_16m
cd ..
make
```

### 2. Bringup tun interface on Host
.... SNIP ....

### 3. Run QEMU with netwrok interface on Host
```
cd $TIZENRT_BASEDIR/os
make download
```

### 4. Bringup interface on QEMU
... SNIP ...

### 5. Bringdown interface on Host
... SNIP